portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -89,6 +89,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // if we want to add comment, then there's no commentID
         if ("add".equals(action)) {
             String text = request.getParameter(Comments.TEXT);
+            if (text == null) {
+                response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Parameter 'text' doesn't exist, required for action 'add'");
+                return;
+            }
+
```

Comment:
That does not seem related to adding the map section.
Two unrelated things should be done in two different commits.

Reply:
+1. Just to be extra clear, we mean a separate Pull Request.